server/prisma/migrations/20240210185042_add_refund_constraint/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- This is an empty migration.
+ALTER TABLE "refunditems" ADD CONSTRAINT "refund_ticketitems_check" CHECK (("donationid_fk" is null AND "order_ticketitemid_fk" is not null) OR ("order_ticketitemid_fk" is null AND "donationid_fk" is not null));
```

Comment:
Does this need to be a separate migration? The comment should also be removed or altered

Reply:
Currently Prisma does not support constraints so the refund constraint needs to be added manually. I added it to the init migration, in the future we just need to remember to manually add it back if we ever get rid of the init migration.

Reply:
Ah interesting that prisma does not support contraints. The idea with this is that we won't have to start over the migrations and once we go into production, we won't be able to